src/components/filmDetailsPage/Page.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="main-block">
+    <div class="firs-block">
+      <FilmInfo
+        imageSrc="https://www.vintagemovieposters.co.uk/wp-content/uploads/2020/12/IMG_1021-scaled.jpeg"
```

Comment:
Consider keeping all such specific props as a single "data" / "info" prop - object when you keep all the data specific fields.

Additionally, once you've grouped all the movie deatils in an object and moved to "data()" option, consider creating "src/core/data" where you're going to keep all the mock data. 
The idea is to have a central place for all the mock data to be able to easily reuse them in Components, stories, tests, etc.

Reply:
In the next PR I moved all the data to the store.